README.md

```diff
@@ -2,29 +2,74 @@
 
 A bookmarklet for monitoring GPT ad units and their associated targeting on web pages.
 
+* [Dev Requirements](#dev-requirements)
 * [Installation](#installation)
+* [Development](#development)
+* [Charles Setup](#charles-setup)
```

Comment:
I think we should create a separate readme for Charles Setup since it may or may not be used to develop this application

Reply:
maybe that can be in the Wiki and ill link the Wiki link in readme?